doc/api/tls.md

```diff
@@ -895,7 +895,7 @@ use only `tls.connect()` to upgrade the socket:
 
 ```js
 const tls = require('tls');
-const secureSock = tls.connect({ port: 6697, host: 'irc.freenode.net' }, () => {
+const secureSock = tls.connect({ port: 6697, host: 'https://example.org:443/' }, () => {
```

Comment:
I am sorry if we were too vague about this, but this is not how TLS sockets work. You need to specify `example.org` as the host and `443` as the port instead of `6697`. `https://example.org:443/` is a URL, @addaleax just used it to point out that *browsers* can connect to port 443. Sockets are below HTTP level, HTTP(s) is built on top of (TLS) sockets.